server/pageParser.js

```diff
@@ -1,10 +1,13 @@
 const SnippetLoader = require('./snippetLoader');
 const DomLoader = require("./domLoader");
 const Aeternity = require("./aeternity");
+let logger = require("./logger")(module);
 
 module.exports = class PageParser {
 
-  constructor(aeternity = null) {
+  constructor(aeternity = null, contextInfo) {
```

Comment:
required arguments should come before optional IMO

Reply:
it's not required, I added a default null, I wouldn't like to put it before the aeternity client argument logically